mlflow/protos/databricks_model_artifacts_pb2.py

```diff
@@ -0,0 +1,287 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Is this autogenerated?

Reply:
Yep, running `generate-protos.sh` autogenerates all `*_pb2.py` files from the `*.proto` files.